contracts/vortex/CarbonVortex.sol

```diff
@@ -183,6 +183,28 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         _allocateRewards(msg.sender, tokens, rewardAmounts);
     }
 
+    /**
+     * @dev withdraws funds held by the contract and sends them to an account
+     *
+     * requirements:
+     *
+     * - the caller must be the admin of the contract
+     */
+    function withdrawFunds(
```

Comment:
I suggest adding a batch function, to save gas when this is being invoked by an external sweeper contract

Reply:
discussed and decided that for the time being it's unnecessary and that it will be added to a generic Vault contract later on